cre.cpp

```diff
@@ -3545,6 +3549,19 @@ static int getImageDataFromPosition(lua_State *L) {
     return 0;
 }
 
+static int smoothScaleBlitBuffer(lua_State *L) {
```

Comment:
Besides a comment you could also consider giving it a less safe name? (Just a thought; I'm fine with just the comment.)

Reply:
(Well, with a less safe name, if we end up one day using it :) I'd have to make a PR here to rename it. With this safe pretty name, we can just use it and pretend I forgot to mark that comment as obsolete :)